docs/dxp/latest/en/site-building/personalizing-site-experience/experience-personalization/personalizing-collections.md

```diff
@@ -8,20 +8,28 @@ By default, any user can see all the Collection's content and is part of *Anyone
 To customize a Collection using Segments, you must define the Segments first. For more information, see [Creating and Managing User Segments](../segmentation/creating-and-managing-user-segments.md).
 ```
 
-![Collections are available for anyone, by default](./personalizing-collections/images/02.png)
+![Collections are available for anyone, by default](./personalizing-collections/images/01.png)
 
 Consider the following example. You want to increase sales in your online Kitchenware Store by offering exclusive promotions to registered users. You create a Collection including promotions for registered users and news about your products. You want all users to see the news about your products, but you want to restrict the promotions to registered users. In this example, you can create a new Segment for registered users, and link the Segment to a new Personalized Variation that filters the promotional content.
 
 ```{tip}
 You can create multiple Personalized Variations for a Collection, and edit or delete them as needed. You can also edit the *Anyone* Personalized Variation, but you cannot delete it.
 ```
 
+## Prioritizing Variations
+
+> Availability: Liferay Portal 7.4
+
+The Collections variations offer a way to prioritize which variation should be rendered on your site to users belonging to more than one segment. You have access to a sortable list for the Personalized Variations in the Collection configuration page. You can order the list by clicking on the *Actions* button (![Actions Button](../../../images/icon-actions.png)) and choosing to prioritize/deprioritize or you can drag and drop.
```

Comment:
"You can order the list by clicking on the *Actions* button (![Actions Button](../../../images/icon-actions.png)) and choosing to prioritize/deprioritize or you can drag and drop."

This has the right essential structure and info, though it could be smoothed out a little. Though I might go with something like, 
"You can change the rank of a variation by clicking its *Actions* button (![Actions Button](../../../images/icon-actions.png)) and selecting *Prioritize* or *Deprioritize*. Alternatively, you can drag and drop variations to create the desired hierarchy."